solaris/src/wei.rs

```diff
@@ -0,0 +1,82 @@
+use types::U256;
+
+lazy_static! {
+    pub static ref TETHER: U256 = U256::from(10).pow(30.into());
+    pub static ref GETHER: U256 = U256::from(10).pow(27.into());
+    pub static ref METHER: U256 = U256::from(10).pow(24.into());
+    pub static ref KETHER: U256 = U256::from(10).pow(21.into());
+    pub static ref ETHER: U256 = U256::from(10).pow(18.into());
+    pub static ref FINNEY: U256 = U256::from(10).pow(15.into());
+    pub static ref SZABO: U256 = U256::from(10).pow(12.into());
+    pub static ref GWEI: U256 = U256::from(10).pow(9.into());
+    pub static ref MWEI: U256 = U256::from(10).pow(6.into());
+    pub static ref KWEI: U256 = U256::from(10).pow(3.into());
+}
+
+pub fn from_tether<T: Into<U256>>(v: T) -> U256 {
```

Comment:
Could you also add docs while you are here? We should later add `#!deny(missing_docs)` to the whole crate.

Reply:
sure